mobile/cmp/error/ErrorMessage.ts

```diff
@@ -4,36 +4,52 @@
  *
  * Copyright © 2023 Extremely Heavy Industries Inc.
  */
-import {div, frame, p} from '@xh/hoist/cmp/layout';
-import {hoistCmp, HoistProps, PlainObject} from '@xh/hoist/core';
+import {div, filler, frame, hbox, p} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistProps} from '@xh/hoist/core';
 import {button, ButtonProps} from '@xh/hoist/mobile/cmp/button';
 import '@xh/hoist/mobile/register';
-import {isEmpty, isNil, isString} from 'lodash';
+import {isNil, isString} from 'lodash';
 import {isValidElement, ReactNode, MouseEvent} from 'react';
 
 import './ErrorMessage.scss';
+import {Icon} from '@xh/hoist/icon';
 
 export interface ErrorMessageProps extends HoistProps {
     /**
      * If provided, will render a "Retry" button that calls this function.
      * Use `actionButtonProps` for further control over this button.
      */
     actionFn?: (e: MouseEvent) => void;
+
     /**
      * If provided, component will render an inline action button - prompting to user to take some
      * action that might resolve the error, such as retrying a failed data load.
      */
     actionButtonProps?: ButtonProps;
+
+    /**
+     * True to provide a link to display additional details to user in a popup.
+     * Use `detailsButtonProps` for further control over this button.  Default false.
+     */
+    detailsFn?: (error: unknown) => void;
```

Comment:
Same comment as above, docstring suggests `detailsFn` accepts a boolean, but signature and implementation expect a closure